src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -100,6 +101,14 @@ static DatadogLogging()
                 }
 
                 SharedLogger = loggerConfiguration.CreateLogger();
+
+                // Log some information to correspond with the app domain
+                SharedLogger.Debug(
+                    "OSArchitecture: {0}, OSDescription: {1}, ProcessArchitecture: {2}, FrameworkDescription: {3}",
+                    RuntimeInformation.OSArchitecture,
```

Comment:
I'm too late to the party, but we probably should have used [`FrameworkDescription`](https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/FrameworkDescription.cs). It reports more useful version numbers and reduces the reliance on an external dll in .NET Framework.

Reply:
Good point, I didn't think of that